.github/ISSUE_TEMPLATE/rule_bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Rule bug report
 about: Report an issue with a specific rule
 title: ''
-labels: ''
+labels: 'bug'
```

Comment:
I think external users won't have permission to auto-attach this label, just FYI

Reply:
Weird. It's one of the defaults that come with new GH repos; We'll find out 🤷‍♂️